### ISSUE TITLE
Implementing dectofrac feature

### ISSUE THREAD

**Issue by mcreng** [danvim/ProcalParsing#19]

**Comment by mcreng** [comment]
After this we can fix #5 properly

**Comment by mcreng** [comment]
Implemented in commit https://github.com/danvim/ProcalParsing/commit/b7f031da9d1c2ba29e175d80812d4e8cf672c850.
Example here:
```java
BigFraction a;
//Two types of constructor
//1. BigFraction a = new BigFraction(BigDecimal), which converts BigDecimal into BigFraction, always keep numerator negative
//2. BigFraction a = new BigFraction(int, int), which gives a the numerator and denominator according to the parameters directly, always keep numerator negative
//a.Numerator() returns numerator of a in BigDecimal
//a.Denominator() returns denominator of a in BigDecimal
